src/injector/index.js

```diff
@@ -7,36 +7,34 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree. 
 */
+"use strict";
 
-const config = require('./config');
```

Comment:
The whole config must be supplied to the BD module. https://github.com/Jiiks/BetterDiscordApp/pull/472/files#diff-d926edcb4918f619b0b8c1a487b4807bR37